packages/actions/test/unit/security.test.ts

```diff
@@ -188,6 +260,115 @@ describe("Security", () => {
         })
     })
 
+    if (envType === TestingEnvironment.PRODUCTION) {
+        // Tests related to multi part upload security
+        // @note We want to make sure that a contributor can
+        // 1. only upload when in contributing status (current contributor)
+        // 2. only upload a file with the correct name
+        // 3. only upload a valid zkey file and previous zkey
```

Comment:
Why `...and previous zKey`?

Reply:
had the wrong assumption when started designing this and never removed it from the comments🙃